docs/src/components/layouts/utility-page-layout/utility-page-module-display.jsx

```diff
@@ -23,11 +23,12 @@ const UtilityPageModuleDisplay = ({
 	utility_examples = null,
 }) => {
 	// Get module information or draw an alert.
-	const utilityInfo = utility_module_name
-		? utilityModules[utility_module_name]
-		: null;
+	const utilityInfo =
+		utility_module_name && utility_module_name !== 'table-display'
```

Comment:
The `table-display` stuff should be removed from here. Only utility modules should use the utility_module namespace.

If you would like to propose a` tokens` section of the utility-page-module-display I can be open to that suggestion.